libs/sdk-ui-pivot/src/impl/structure/tableDescriptorTypes.ts

```diff
@@ -0,0 +1,275 @@
+// (C) 2007-2021 GoodData Corporation
+import { IAttributeDescriptor, IResultAttributeHeader } from "@gooddata/sdk-backend-spi";
+import { DataSeriesDescriptor, DataSeriesId } from "@gooddata/sdk-ui";
+import { ColDef, ColGroupDef, Column } from "@ag-grid-community/all-modules";
+
+export const ColumnGroupingDescriptorId = "groupingDescriptor";
+
+/**
+ * Represents table column which displays data slice attribute element names. PivotTable interface also calls
+ * these `attribute column`.
+ *
+ * In other words, in a table that is defined as (Row: A1, Measures: M1) and looking like:
+ *
+ * A1    | M1
+ * ----------
+ * val1  | 1
+ * val2  | 2
+ *
+ * There is one SliceCol - for row attribute A1. In this table, there will be one row for each
+ * element of attribute A1.
+ *
+ * TODO: rename to attributeCol?
+ */
+export type SliceCol = {
```

Comment:
If the "slice" is the final name then change the "rowHeaderDescriptor" to something more slicier :)

Reply:
it's not final name. i will revisit the naming in subsequent PR, more names need to change.. even the DataCol* stuff